### PR TITLE
[Azure.AI.TextAnalytics.Authoring] Standalone package for the authoring client

### PR DESCRIPTION
**Pull request by LouaiZahran**
As per our agreement, we decided to publish the TA authoring client as a standalone beta package until we have the chance to consolidate both the authoring and runtime packages into one package.

### Remaining work:
Recording the tests.

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### Features Added
+
```
Comment (heaths) [inline sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/CHANGELOG.md:6]: Should at least have `- Initial release` or something for at least one entry ready when released.

File: sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/Directory.Build.props [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    Add any shared properties you want for the projects under this package directory that need to be set before the auto imported Directory.Build.props
+  -->
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
```
Comment (heaths) [inline sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/Directory.Build.props:5]: File not necessary. May as well delete.

File: sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+# Azure Authoring client library for .NET
+
+This section should give out brief introduction of the client library.
```
Comment (heaths) [inline sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/README.md:3]: The README needs to be updated at some point. Do you have a separate tracking issue?

File: sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/samples/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+products:
+# Including relevant stubs from https://review.docs.microsoft.com/help/contribute/metadata-taxonomies#product
+- azure
+name: Azure.AI.TextAnalytics.Authoring samples for .NET
+description: Samples for the Azure.AI.TextAnalytics.Authoring client library.
+---
+
+# Azure.AI.TextAnalytics.Authoring Samples
+
+<!-- please refer to <https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/samples/README.md> to write sample readme. -->
```
Comment (heaths) [inline sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/samples/README.md:14]: This needs to reference every sample herein.

**Review by heaths — CHANGES_REQUESTED** [review]
One main question: all the work you did here could easily have been done in the existing package. We don't have separate authoring packages for the other Language clients, but you can look at their autorest.md for how I do a batch-build to reference separate input files (if needed). Why release a separate package when even less work is required than this to add it to the existing package?

Who was this discussion about releasing a separate package with?

**Comment by LouaiZahran** [comment]
> One main question: all the work you did here could easily have been done in the existing package. We don't have separate authoring packages for the other Language clients. Why release a separate package when even less work is required than this to add it to the existing package?
> 
> Who was this discussion about releasing a separate package with?

This discussion was done a while ago (almost a month ago) in a long email thread titled "Authoring Client Status". I'm not sure whether you were included but Kareem, Krista, and Johan were there, and they came to the conclusion that currently it is better to have a separate package for the authoring clients for both Python and .NET as they had concerns about the maintainability of the authoring client in the same package as the runtime one.

Unfortunately, my internship has ended and I no longer have access to my work email to forward the thread to you. The SDK is handed over to @AhmedLeithy to continue working on and you can talk to Kareem or Krista to have a better understanding of the situation.

**Comment by heaths** [comment]
Thanks. I'll take this offline with them.

**Pull request closed without merging**